### DISCUSSION TITLE
What the hell is a "photonic gas"??

### DISCUSSION THREAD

**Discussion by Pierre314159** [a-cosmology-group/acg#255] [Gravitation Models]
As an experimentalist, what do you think of this experiment?

http://dx.doi.org/10.13140/RG.2.2.33179.95522

It almost looks too good to be true...

**Comment by LifeIsStrange** [comment]
great question, another theory that represent gravity as a difference of pressure/density of a background medium/particle is
https://www.gsjournal.net/Science-Journals/Essays-Gravity/Download/8377
it has the merit to involve neutrinos, which do interact with matter and might explain such effect though nobody has made a quantitative analysis

**Comment by Francois-Zinserling** [comment]
These are my opinions.
Photonic gas = sea of photons, much like Dirac's particle sea, except with photons which cannot 'stand still'.
Photonic gas can also be a descriptive name for a quantum field.
He might mean aether but he is avoiding the use of the word.

I am a strong supporter of the Fatio / Le Sage model. 
As he says in the paper the biggest obstacle is explaining where the energy goes to (even after you presume the sea of photons exist to start off with), but I am disappointed by his response that it goes into mass. Although @HanDeBruijn might like it a bit more.

I'm commenting about the original link by @Pierre314159 , although I am also aware of the work of John-Erik Persson.

**Reply by budrap00** [reply]
"Photon gas" is just quantum babble - typical of an inability to keep the categorical distinction between matter and electromagnetic radiation straight. The experiment itself just seems to demonstrate that gamma rays partially penetrate the material bodies used in the experiment. Assuming that there is some ambient gamma radiation in the vicinity of the experiment then in the immediate vicinity of the material bodies some or all of that radiation would pass through the bodies but some or all would be be absorbed. In either case the result would be a deficit near the bodies. 

It is an interesting but not surprising result - that's more or less how medical x-ray technology works - partial penetration. Turning a highly-constrained table top experiment result into the causal mechanism for gravity though doesn't make a whole lot of sense, especially since it inverts the electromagnetic density gradient that is observed (and theorized) at a gravitating body.

**Reply by HanDeBruijn** [reply]
@Francois-Zinserling writes:
>I am disappointed by his response that it goes into mass. Although @HanDeBruijn might like it a bit more.

Quote from the article: _in our case, the increase in the mass of the detector_ $\Delta M$ _during time_ $T=1000\mbox{ }year$ _[ .. ] will be only_ [ .. ] $\approx 10^{-4}\mbox{ }kg$ . We could check that with the Variable Mass Theory, of course, if we only knew what the original mass $M$ of the detector is.

$$
\frac{M+\Delta M}{M} = \left(1+\frac{T}{A}\right)^2 \quad \Longrightarrow \quad \frac{\Delta M}{M} \approx \frac{2T}{A}
$$

where $A=$ age of the earth. Numerically: $\Delta M = M.2\times 1000/(4.543\times 10^9) \approx 10^{-4}\times 0.0044 M$

**Comment by RedshiftDrift** [comment]
A photonic gas is a group of photons that are constantly scattering off a medium instead of propagating on a straight line.  When it's very *very* foggy outside and all you see around you is 'white', you are standing in a photonic gas.  Now raise your hand in front of your face and you will see a darker region in the direction of your hand, indicating photon absorption by your hand.

So @Pierre314159 you could do Samokhvalov's experiment at home with a basketball, a photometer and a lot of fog. (Or with ball bearings in a glass of milk...)

Samokhvalov's is *a trivial experiment* done with expensive gamma-ray detectors.
What's too good to be true?

The experiment uses the atmospheric background gamma radiation from decay of radioactive elements in the atmosphere and cosmic radiation.  The force due to absorption in steel is very small.  He calculates that to get a force comparable to Newton's gravity one needs to shorten the wavelength of light (increase its energy) by 14 orders of magnitude, and comments that "В результате произойдёт рост мощности гамма радиации до 10^16 W" which would produce the instantaneous evaporation of the steel ball.

I agree, $10^{16}$ W would evaporate a steel ball very quickly.

Since Samokhvalov was able to do the experiments, this shows that steel balls don't evaporate, and once again this falsifies the whole idea of gravity being due to absorption of particles.

**Reply by budrap00** [reply]
That's what I meant by a categorical distinction. A fog is a gas of water vapor, the electromagnetic radiation is scattered off the discreet water vapor molecules producing a diffuse glow of electromagnetic radiation that has been well understood in terms of wave behavior for centuries. The quantum scale concept of a photon adds nothing meaningful to that picture, only ontological confusion. The fog _is_ a gas composed of three dimensional particles; electromagnetic radiation _is not_ a gas because it is electromagnetic radiation and it _is not_ composed of three dimensional particles.

On classical scales and above electromagnetic radiation is a wave phenomenon. On those scales quantum behavior is irrelevant in the same way that our behavior is irrelevant on cosmological scales. "Photon gas" is a physically superfluous concept.

**Comment by LifeIsStrange** [comment]
I want to thank you for reintroducing me to Lesage/Fatio theory, it seems to be the most physical explanation of gravity and the literature is a goldmine of major results and anomalies.
This theory of gravity differs in many ways from most which makes it testable experimentally.
One of the most important divergence is the concept of gravitational shielding

It explains among other things:
magical reduction of weight in superconductors
https://www.sciencedirect.com/science/article/abs/pii/092145349290055H
time variation on earth
https://www.scirp.org/journal/paperinformation?paperid=118417
the Allais effect
https://ui.adsabs.harvard.edu/abs/2017AAS...22914006M/abstract
https://www.academia.edu/65481036/The_Allais_Effect_Coincidence_Between_Newtonian_and_Lesagian_Gravity

One thing that is unclear to me is wether Lesage gravity works if we assume the graviton is scattered (reflected) or slowed versus absorbed by a massive object.

https://phys.org/news/2016-02-year-old-ether-cosmological-temperature-gradient.html

**Reply by Francois-Zinserling** [reply]
Gravitational shielding is one of the arguments against Fatio/Le Sage, in that it is not observed. The linked documents look promising and I have some reading to do. I also propose that orbital velocities in galactic disks are due to gravitational shielding, which is the hint at 'the other hidden matter' as one paper refers to it. Thus no dark matter required.

>One thing that is unclear to me is wether Lesage gravity works if we assume the graviton is scattered (reflected) or slowed versus absorbed by a massive object.

Feynman proved that the mechanism cannot work on reflection. This must account for direct reflection or scattering. 
'Slowing' is proposed as a possible mode change, but through continuous interactions, space will eventually be filled with 'slow' corpuscles and gravity will start to fail. Not observed. Same was for Fatio's physical pellet/ball corpuscles, which must eventually clog up all atoms and gravity would fail.

There has been a lot of discontent with the Fatio/Le Sage model, further fueled by the greats such as La Place, Maxwell, Feynman, Poincare, and more others recently. Unfortunately, their heavy-weight rebuffs have caused many others to discount this model outright.
For me as an engineer it is the only logical explanation of the mechanics of gravity, and I am pursuing it with a passion. Not a real chance that I could provide a break-through, but if I can just add that fraction of momentum ...

Many of the original objections have been discounted through the advances in science, e.g. Fatio/Le Sage had no idea of photons or atoms (being mostly empty space). 

Some valid objections remain. Among them 
 - the penetration ability of the 'corpuscles'. Even though atoms are mostly empty space, we know today that not even photons or x-rays have that kind of penetration ability. In the linked papers reference is made how 'low' the shielding of the moon is. That means the portion of Fatio/Le Sage particles/corpuscles interacting in gravity must be very small compared to the total count that 'goes right through', so that we do not even notice the 'missing count' from direction of the moon. 
 - Energy. If absorption is part of the mechanism, the absorbed energy is certain to 'blow things up'. This notwithstanding that no other model can explain how the same amount of energy must be 'radiated', or where the energy is sourced to 'curve space'. However, for the Fatio/Le Sage model, this one deals it a deathblow. A solution to the energy problem is required.

**Reply by ExpEarth** [reply]
I would add that for these reasons the Le Sage model is not likely to be confirmed in the way that it has traditionally been visualized, as an absorption or scattering of waves or tiny particles by masses.  Proof of shielding is hard to get (but Quirino Majorana must have found _something_ .  Over the years I have migrated to a 'quasi Le Sage' model, where the absorption of light is not at the particle centres, but in the filaments of spacetime which connect all particles having mass, e.g., as in #204
